docs/organizations/using-a-coding-standard.md

```diff
@@ -86,8 +84,6 @@ To edit an existing coding standard or change the repositories that follow that
 
 1.  Click the button **Save and apply standard** on the repository selection page to save your changes to the coding standard.
 
-    {% include "../assets/includes/coding-standard-apply-important.md" %}
```

Comment:
We should also delete the included file `docs/assets/includes/coding-standard-apply-important.md` since it seems like after these changes it will no longer be used. :broom:

Reply:
Yes, I was thinking of doing that, but I wanted to validate with you.